app/models/event.rb

```diff
@@ -14,6 +14,7 @@ class Event < ApplicationRecord
   has_many :users, through: :inscriptions
   has_and_belongs_to_many :tags
   has_one_attached :image
+  has_many :preferences, through: :user_event_preference
```

Comment:
mmmhhh, does this sound a bit weird to you as it does for me? Does an event have preferences? Maybe an event belongs to a user's preferences through the `user_event_preference` table?